src/feature/history/ui/history.document.ts

```diff
@@ -0,0 +1,21 @@
+import { DocumentDto } from '@earnkeeper/ekp-sdk';
+
+export class HistoryDocument extends DocumentDto {
+  constructor(properties: HistoryDocument) {
+    super(properties);
+  }
+   readonly id: string;
+   readonly created_date: Date;
```

Comment:
Change these variables to use lowerCamelCase

Reply:
got this